docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='optimizer-download-proc']
+= Download {PLANNER}
+
+{PLANNER} is production ready.
+The API is stable but backward incompatible changes can occur.
+
+.Prerequisite
+The {PRODUCT} has been downloaded and installed from the https://access.redhat.com[Red Hat Customer Portal]. See Installing Decision Manager for more information about installing {PRODUCT}.
+//@doc-link:Add link to Installation assembly above. 
+
+.Procedure
+. Download the {PLANNER} file from the https://access.redhat.com[Red Hat Customer Portal].
```

Comment:
The prerequisite says it needs to be installed alread, but this procedure says that it will be downloaded. Might the prereq be obsolete?

Reply:
yep, removed.